backend/src/crd/controller/scheduledworkflow/util/label.go

```diff
@@ -15,7 +14,7 @@ func GetRequirementForCompletedWorkflowOrFatal(completed bool) *labels.Requireme
 	if completed == true {
 		operator = selection.Equals
 	}
-	req, err := labels.NewRequirement(common.LabelKeyCompleted, operator,
+	req, err := labels.NewRequirement("Completed", operator,
```

Comment:
why not keep using `common.LabelKeyCompleted` here?

Reply:
ah, I see, this if from the `argo` library. Does the Kubeflow or Kubernetes code use constants or plain strings as well? I see Tekton seems to use plain strings but that leave some room for typos causing bugs down the road

Reply:
This label supposes to come from Argo, but in Tekton it doesn't have the concept of Argo phases, so I was pointing it to some other label. I think there's another label owned by KFP that behaves the similar way, maybe I can switch to that one so we don't have to depend on a string input.

Reply:
I updated the label to `LabelKeyWorkflowPersistedFinalState` which is what KFP use to tag for pipelines that are completed.